README.md

```diff
@@ -49,3 +49,92 @@ Apache header:
     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
     See the License for the specific language governing permissions and
     limitations under the License.
+
```

Comment:
While we haven't checked in our BUILD file to resultstore/v2, can we add the documentation on how to download and populate the generated python file too?